project/assets/test/ui/chat/chat-full.chat

```diff
@@ -3,4 +3,5 @@
 boatricia: ^_^ Oh! I remember you from before. I'm Boatricia, I'm sort of new here and trying to make some friends.
 player: Nice to meet you! People here call me #player#.
 player: /._. Well! ...If you're trying to make some friends, I'll try to cook you something special okay? Then maybe you can be my friend.
+// Comment giving this chat some context
```

Comment:
Consider a less opaque comment like `// Example comment`. The phrase 'comment giving this chat...' can be interpreted as a request or a plan to finish something later.

Reply:
Replaced with `example comment`